matlab/schemas/+preprocess/ExtractRaw.m

```diff
@@ -29,7 +28,7 @@
         
         function makeTuples(self, key)
             
-            %% extract masks and traces for Galvo Scanner
+            % extract masks and traces for Galvo Scanner
```

Comment:
Can we leave the %% in the future. They are useful to visually organize the code in the Matlab editor.